tests/bluechi_test/container.py

```diff
@@ -157,3 +157,15 @@ def enable_unit(self, node_name: str, unit_name: str) -> None:
         result, output = self.exec_run(f"bluechictl enable {node_name} {unit_name}")
         if result != 0:
             raise Exception(f"Failed to enable service {unit_name} on node {node_name}: {output}")
+
+    def run_python(self, python_script_dir: str, python_script_filename: str) -> \
```

Comment:
Why do you separate between the directory and the file?

Reply:
Considering  your response [here](https://github.com/containers/bluechi/pull/485#discussion_r1299700232), you don't need the name of the file, you can just use a randomly generated name, no?

Reply:
Actually, using a generated name is safer in case of concurancy

Reply:
Added it. Could you have another look? @ygalblum

Reply:
Great for the transient script file. Now, we can combine the two parameters into one, no?

Reply:
Fair point. Changed :heavy_check_mark: